test/main.test.js

```diff
@@ -492,7 +492,7 @@ describe('bundling single js feed', () => {
         supertest(server)
             .post('/bundle/js')
             .send(['completelyfake.json'])
-            .expect(409));
+            .expect(404));
```

Comment:
revert these?

Reply:
404 seems better than 409 though right?

Reply:
if it's _just_ for stuff not found, I agree 🙂